networking/src/main/java/ar/com/wolox/wolmo/networking/retrofit/handler/NetworkRequestHandler.kt

```diff
@@ -0,0 +1,60 @@
+package ar.com.wolox.wolmo.networking.retrofit.handler
+
+import retrofit2.Response
+
+/**
+ * An adapter that converts Retrofit's responses to other, more specific, ones
+ * depending on network status codes or failures when using Kotlin's Coroutines.
+ *
+ * @param <T> the type of object expected to be returned from the API call
+ */
```

Comment:
I suggest to use this convention https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments . We are already using this way of creating comments in `wolmo-core`.

Reply:
The information in the comments is perfect, you just need to adapt them to the convention